src/components/plots/plot.ts

```diff
@@ -23,6 +23,11 @@ module Plottable {
       accessor: Accessor<any>;
       scale?: Scale<D, R>;
     }
+
+    export class Animator {
```

Comment:
We should probably standardize around what we want to do for `string`y enums. I think it'd be better if this was an exported `static var` instead of a `class`.